modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -10195,6 +10449,11 @@ public void testPutCustomObjectEntryWithNestedCustomObjectEntriesInOneToManyRela
 			_OBJECT_FIELD_NAME_2, _NEW_OBJECT_FIELD_VALUE_1);
 	}
 
+	@Test
+	public void testPutScopeScopeKeyByExternalReferenceCode() throws Exception {
+		_testPatchPutScopeScopeKeyByExternalReferenceCode(Http.Method.PUT);
+	}
```

Comment:
Can we merge both tests?
We have a similar example in this class, check the method `testPatchPutCustomObjectEntryByExternalReferenceCodeWithAttachmentObjectField`


```
@Test
public void testPatchPutScopeScopeKeyByExternalReferenceCode() throws Exception {
    _testPatchPutScopeScopeKeyByExternalReferenceCode(Http.Method.PATCH);
    _testPatchPutScopeScopeKeyByExternalReferenceCode(Http.Method.PUT);
}
```